src/main/groovy/gl51/project/store/Product.groovy

```diff
@@ -0,0 +1,12 @@
+package gl51.project.store
+
+class Product {
+    int id
+    String name
+    String description
+    double price
+    double idealTemperature
+    int getID(){
```

Comment:
I think that id is public in this case so you shouldn't need a getID()

Reply:
Groovy does that for you, no dumb getter/setter in java. 
you can write them, but if you do nothing, you get the same behaviour from groovy
http://groovy-lang.org/style-guide.html#_getters_and_setters